frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -156,10 +156,10 @@
                                                         vm.stopLoader();
                                                     },
                                                     onError: function(response) {
-                                                        if (response.data['detail']) {
+                                                        if (response.status == '404') {
                                                             var error = "Please select a team first!";
-                                                        } else if (response.data['error']) {
-                                                            error = response.data['error'];
+                                                        } else {
+                                                            error = "Server error";
```

Comment:
@codervivek: I am curious to know that what is the value of response.status in the else condition. Can you please paste the value here?

Reply:
@deshraj Actually response.data returns a HTML string of Django page. Since response.data['detail'] didn't had any value, I checked for response.status. Since onError will only execute when there some error so I have only checked for 404 or else there I have classified them as "server error". Please suggest if you want to add some other case.